<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Transmission</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_sendCtrl">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт передачи</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_log">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>391</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_state">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сообщения</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_nav">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <property name="title">
     <string>NAV</string>
    </property>
    <widget class="QPushButton" name="pushButton_recordCtrl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Старт запись</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>311</width>
       <height>366</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_roll"/>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEdit_verticalSpeed"/>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_aerialSpeed"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_latitude">
        <property name="text">
         <string>Широта</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEdit_aerialAltitude"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_longitude">
        <property name="text">
         <string>Долгота</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_aerialAltitude">
        <property name="text">
         <string>Барометрическая высота</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_aerialSpeed">
        <property name="text">
         <string>Воздушная скорость</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_latitude"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_pitch"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_gpsSpeed">
        <property name="text">
         <string>Путевая скорость</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_gpsSpeed"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_gpsCourse">
        <property name="text">
         <string>Путевой угол</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_pitch">
        <property name="text">
         <string>Тангаж</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_yaw">
        <property name="text">
         <string>Рысканье</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_longitude"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_verticalSpeed">
        <property name="text">
         <string>Вертикальная скорость</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_gpsCourse"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_roll">
        <property name="text">
         <string>Крен</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_altitude">
        <property name="text">
         <string>Высота</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_altitude"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_yaw"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>331</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_ip">
       <property name="text">
        <string>Ip адрес</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_ip"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_port">
       <property name="text">
        <string>порт</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_port"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
